components/sections/Header.tsx

```diff
@@ -1,34 +1,42 @@
-import React from "react";
-import { chakra, Flex, Stack } from "@chakra-ui/react";
+import React, { ReactElement } from "react";
+import { Box, Flex, Stack } from "@chakra-ui/react";
 import { ButtonPrimary } from "@/components/core/Button";
-import { LogoWhite } from "@/components/core/Branding";
-
+import { LogoWhite, LogoBlack } from "@/components/core/Branding";
 import SearchInput from "@/components/core/SearchInput";
 
-const HeaderButton = chakra(ButtonPrimary, {
-  baseStyle: {
-    bg: "black",
-    _hover: { bg: "gray" },
-  },
-});
+interface HeaderProps {
```

Comment:
can you add shadow variable? On the search bar I added that the white header has a shadow :)

Reply:
Will the white header always have a shadow? Just wondering if I need to add another prop

Reply:
All of the headers should have a shadow